lib/logstash/outputs/elasticsearch.rb

```diff
@@ -458,10 +458,8 @@ def submit(actions)
   public
   def flush(actions, teardown = false)
     begin
-      @logger.debug? and @logger.debug "Sending bulk of actions to client[#{@client_idx}]: #{@host[@client_idx]}"
       submit(actions)
     rescue => e
-      @logger.error "Got error to send bulk of actions to elasticsearch server at #{@host[@client_idx]} : #{e.message}"
```

Comment:
Yeah but we are missing the context around it which is in the message "Got error to send bulk of actions to elasticsearch server" .. better to see that than a stack trace with 
```
Failed to flush outgoing items {:outgoing_count=>26, :exception=>#<NoMethodError: undefined method[]' for nil:NilClass>, :backtrace=>["/opt/logstash/vendor/bundle/jruby/1.9/gems/logstash-output-elasticsearch-0.2.4-java/lib/logstash/outputs/elasticsearch.rb:464:in flush'", "/opt/logstash/vendor/bundle/jruby/1.9/gems/stud-
```